src/renderer/browser/components/DAppContainer/DAppContainer.js

```diff
@@ -23,17 +25,18 @@ export default class DAppContainer extends React.PureComponent {
     dequeue: func.isRequired,
     empty: func.isRequired,
     openTab: func.isRequired,
-    closeTab: func.isRequired
+    closeTab: func.isRequired,
+    onFocus: func // eslint-disable-line react/no-unused-prop-types
   }
 
   static defaultProps = {
-    className: null
+    className: null,
+    active: false,
+    onFocus: noop
   }
 
   async componentDidMount() {
-    window.addEventListener('focus', this.handleFocus);
```

Comment:
Whoops, I didn't mean to remove this.  Needs to be added back in before merging.